include/genn/genn/code_generator/codeGenUtils.h

```diff
@@ -106,44 +101,6 @@ bool regexFuncSubstitute(std::string &s, const std::string &trg, const std::stri
 void functionSubstitute(std::string &code, const std::string &funcName,
                         unsigned int numParams, const std::string &replaceFuncTemplate);
 
-//--------------------------------------------------------------------------
-//! \brief This function writes a floating point value to a stream -setting the precision so no digits are lost
-//--------------------------------------------------------------------------
-template<class T, typename std::enable_if<std::is_floating_point<T>::value>::type* = nullptr>
```

Comment:
I am not sure I can parse this ... is it the same as
```std::enable_if_t<std::is_floating_point<T>::value, int> = 0``` ?
Oh boy - feels like I have gotten totally out of touch with more recent C++ standards.